test-framework/provisioning/roles/system-coverage/files/sitecustomize.py

```diff
@@ -1,13 +1,29 @@
 import atexit
-import os.path
+from pathlib import Path
 
 import coverage
 
 
-if os.path.exists("/etc/SuSE-release"):
-	config_file = "/export/test-suites/_common/sles.coveragerc"
-else:
-	config_file = "/export/test-suites/_common/redhat.coveragerc"
+def get_distro():
+	''' get the distro on any newer systemd based distro (sles12+, centos/rhel 7+) '''
+	os_release = Path('/etc/os-release')
+	if not os_release.exists():
+		raise NotImplemented
```

Comment:
Did you mean `NotImplementedError`?

`NotImplemented` serves a different purpose and should even be returned, not raised. https://docs.python.org/3/library/constants.html#NotImplemented

Reply:
Good catch!  Thanks.